file.go

```diff
@@ -109,8 +110,9 @@ func (c *Client) GetFile(id uint) (*File, error) {
 	return file, nil
 }
 
-// UploadFileFromURL uploads a file to threeplay using the file's URL.
-func (c *Client) UploadFileFromURL(fileURL string, options url.Values) (string, error) {
+// UploadFileFromURL uploads a file to threeplay using the file's URL and
+// returns the file ID.
+func (c *Client) UploadFileFromURL(fileURL string, options url.Values) (uint, error) {
```

Comment:
Why don't we keep using string?

Reply:
@markx the FileID is typed as an uint in other places in the library, so I think it should be consistent. Also, the only way to validate that the response is valid is by parsing it as an uint.